src/_Description.scss

```diff
@@ -0,0 +1,18 @@
+li {
+  list-style: none
+}
+
+.description-box {
+  background-color: rgba( 10, 0, 0, 0);
+  color: #fff;
+  margin: auto;
+  width: 75%;
```

Comment:
Can't wait to see our stylings come together! Kim's ListContainer stylings will hold your descriptions as per our wireframe!